mutant/modules/artic_illumina/start.py

```diff
@@ -52,16 +52,28 @@ def run_case(self, resdir):
         if self.config_artic != "":
             confline = "-C {0}".format(self.config_artic)
 
-        cmd = "nextflow {0} -log {1} run {2} {3}/externals/gms-artic/main.nf -profile {4} --illumina --prefix {5} --directory {6} {7}".format(
-            confline,
-            nflog,
-            workline,
-            self.WD,
-            self.profiles,
-            self.prefix,
-            self.fastq,
-            resultsline,
-        )
+        if nanopore:
+            cmd = "nextflow {0} -log {1} run {2}/externals/gms-artic/main.nf -profile singularity,slurm --medaka --prefix {3} --basecalled_fastq {4} --scheme-directory {5}/externals/gms-artic/primer-schemes/midnight/nCoV-2019/V1 {6}".format(
```

Comment:
scheme-directory is strange. I see it is mentioned in the README at gms-artic, but I cannot find anywhere this "parameter" is used. Or am I simply missing it? If not, I think we need to update schemeRepoURL, schemeDir, scheme and schemeVersion. The last two are already specified in the config files under params. The first two should be added there. Then we can remove the scheme-directory option here.

Reply:
if scheme-directory does not matter, then it will be easy to change primers, since this is where you give information regarding primers

Reply:
did as suggested now